examples/list_pod/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_executable(list_pod main.c)
+
+find_package(${pkgName})

```

Comment:
If the `examples` is a seperate project, it cannot find ${pkgName}

Reply:
Fixed